exchanges/ftx/ftx.go

```diff
@@ -344,7 +344,7 @@ func (f *FTX) GetExpiredFutures(ctx context.Context) ([]FuturesData, error) {
 }
 
 // FundingRates gets data on funding rates
-func (f *FTX) FundingRates(ctx context.Context, startTime, endTime time.Time, pair currency.Pair) ([]FundingRatesData, error) {
+func (f *FTX) FundingRates(ctx context.Context, startTime, endTime time.Time, pair currency.Pair, limit int) ([]FundingRatesData, error) {
```

Comment:
int -> int64?

Reply:
I chose int here to minimise the amount of int64 conversions etc when dealing with `len` of a slice, but I'll reexamine :+1: